src/temp/stories/SearchTabForMobile.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import SearchTabForMobile from '../Project/SearchTabForMobile';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof SearchTabForMobile> = {
+  component: SearchTabForMobile,
+};
+
+export default meta;
+type Story = StoryObj<typeof SearchTabForMobile>;
+
+export const SearchInActive: Story = {
+  args: {
+    active: true,
+  },
+};
```

Comment:
Typo here - this should be `SearchInactive` as `inactive` is one word. This will help storybook split the story name correctly as well, as it is dependent on case.